src/Sylius/Behat/Context/Setup/PromotionContext.php

```diff
@@ -199,6 +199,16 @@ public function thisPromotionHasExpired(PromotionInterface $promotion): void
         $this->objectManager->flush();
     }
 
+    /**
+     * @Given /^(this promotion) can not be applied on discounted products$/
+     */
+    public function promotionCanNotBeAppliedOnDiscountedProducts(PromotionInterface $promotion)
+    {
+        $promotion->setApplyOnDiscounted(true);
```

Comment:
There's a contradiction between this and the step description `**can not** be applied on discounted products`.
```suggestion
        $promotion->setApplyOnDiscounted(false);
```